packages/contracts/deployments/mainnet/OVM_CanonicalTransactionChain.json

```diff
@@ -601,32 +601,32 @@
       "type": "function"
     }
   ],
-  "transactionHash": "0x1c20c2ee279bfa7446fd7117b4b147cb2709f581a8052fa66276df2d1af3467b",
+  "transactionHash": "0x7d26b7cfd3f595bebab4d22ca651feb308c3f95e475a8d2e80f4b06ffefed461",
   "receipt": {
     "to": null,
-    "from": "0x77C4c8f491e5ff22a2C0E18b005c3BA0Fe36c1Bc",
-    "contractAddress": "0x405B4008Da75C48F4E54AA39607378967Ae62338",
-    "transactionIndex": 144,
-    "gasUsed": "2814575",
+    "from": "0xd033f09cB85621F98F7A84C64dB381ac16Eff818",
+    "contractAddress": "0xa88e220c7FC7F0D845D2624a5dF1DfD6874B9a44",
+    "transactionIndex": 102,
+    "gasUsed": "2831026",
     "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
-    "blockHash": "0xf94f495e3ac703206e0edc2e0922da91c418f22a6d0af1ca2b9fab1bd5618219",
-    "transactionHash": "0x1c20c2ee279bfa7446fd7117b4b147cb2709f581a8052fa66276df2d1af3467b",
+    "blockHash": "0xb4aa0366ff1e4f3f49f77d89d9025e4102a8cd2a0b3fd64cd1c1a13b0a7c98f6",
+    "transactionHash": "0x7d26b7cfd3f595bebab4d22ca651feb308c3f95e475a8d2e80f4b06ffefed461",
     "logs": [],
-    "blockNumber": 12207792,
-    "cumulativeGasUsed": "12334867",
+    "blockNumber": 12410807,
+    "cumulativeGasUsed": "8803160",
     "status": 1,
     "byzantium": true
   },
   "args": [
-    "0xd3EeD86464Ff13B4BFD81a3bB1e753b7ceBA3A39",
-    18000000,
-    1200000,
+    "0x668E5b997b9aE88a56cd40409119d4Db9e6d752E",
+    15769999,
```

Comment:
Why these weird config values?

Reply:
Oh damn thought I changed that to `15770000` which is ~half a year.